charts/k8up/templates/grafana-dashboard.yaml

```
@@ -0,0 +1,1204 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: name: {{ include "k8up.fullname" . }}-grafana-dashboard
+  namespace: {{ default .Release.Namespace .Values.metrics.grafanaDashboard.namespace }}
+  labels:
+    {{- include "k8up.labels" . | nindent 4 }}
+    {{- with .Values.metrics.prometheusRule.additionalLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+data:
+  grafana-dashboard-k8up.json: |-
+    {
```

```suggestion
    {{`
    {
```

For escaping all of those brackets so that Helm does not fail (see the counterpart below).

Thanks a lot for for the Dashboard, I already deployed it and it shows helpful values out-of-the-box :-D